addons/calendar/models/calendar_event.py

```diff
@@ -119,6 +119,9 @@ def _default_videocall_location(self):
         that period of time.")
     is_highlighted = fields.Boolean(
         compute='_compute_is_highlighted', string='Is the Event Highlighted')
+    is_organizer_alone = fields.Boolean(compute='_compute_is_organizer_alone', string="Is the Organizer Alone",
```

Comment:
Naming is leading to confusion. Organizer is not alone, it is more linked to all attendees having declined the meeting.